docs/migration/index.md

```diff
@@ -7,8 +7,14 @@ parent:
 description: For chains that were scaffolded with Starport versions lower than v0.19.2, changes are required to use Starport v0.19.2. 
 ---
 
-# Upgrading a Blockchain to use Starport v0.19.2
+# Upgrading a blockchain to use Starport v0.19.2
 
-Starport _v0.19.2_ comes with IBC _v2.0.2_. To migrate your chain scaffold with _v0.19_ or _v0.18_ version of Starport, apply changes introduced in PR [#1975](https://github.com/tendermint/starport/pull/1975/files) to your chain.
+Starport v0.19.2 comes with IBC v2.0.2. 
 
-With _v0.19.2_, contents of `tendermint/spm` moved to Starport's own repo. Upgrade your chain's `go.mod` by removing `tendermint/spm` and adding _v0.19.2_ version of `tendermint/starport`.
+With Starport v0.19.2, the contents of the deprecated Starport Modules `tendermint/spm` repo are moved to the official Starport repo which introduces breaking changes.
+
+To migrate your chain that was scaffolded with Starport versions lower than v0.19.2: 
+
+1. Apply the changes that are introduced in PR [#1975](https://github.com/tendermint/starport/pull/1975/files) to your chain.
+
+2. Upgrade your chain's `go.mod` file to remove `tendermint/spm` and addn the v0.19.2 version of `tendermint/starport`.
```

Comment:
Can we add same mapping ("old import path" -> "new import path") we have in the changelog for this bullet?